documentation/aircraft-commands.md

```diff
@@ -162,12 +162,16 @@ any order is acceptable, as long as the command `hold` comes first.
 _Syntax -_ `AAL123 hold [fixname] [left|right] [leg_time]min` or `AAL123 hold`
 
 ### Exit Hold
```

Comment:
If we are going with  `cancel hold` we should change this to cancel hold as well

Reply:
Sure, we can. cancel/exit/continue/etc are all still available aliases, so I thought it was a fine title for the user command documentation. Just found `.cancelHoldingPattern()` to be a clearer match to the opposite `.initiateHoldingPattern()`, compared to `.exitHold()`.

We can change the documentation though :+1: